extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -172,12 +47,15 @@ export class PgpBlockViewRenderModule {
     });
   };
 
-  public renderContent = async (htmlContent: string, isErr: boolean) => {
+  public renderContent = (htmlContent: string, isErr: boolean) => {
+    /*
+    todo:
```

Comment:
do we really need this flag: `successfully_received_at_leat_one_message`, @tomholub ?
it is used in some debug diagnostics?

Reply:
if you don't see clear use, then you can remove it.